README.md

```diff
@@ -423,7 +423,7 @@ Here is a quick summary of what you should take care of when migrating from `pyt
 
 ### Models always output `tuples`
 
-The main breaking change when migrating from `pytorch-pretrained-bert` to `transformers` is that the models forward method always outputs a `tuple` with various elements depending on the model and the configuration parameters.
+The main breaking change when migrating from `pytorch-pretrained-bert` to `transformers` is that the model's forward method always outputs a `tuple` with various elements depending on the model and the configuration parameters.
```

Comment:
It concerns several models, do you mind changing `model's` to `models'` ?

Reply:
Thanks for pointing that out! I've changed it to "every model's forward method", if that is okay.